cirq-core/cirq/ops/pauli_string.py

```diff
@@ -726,6 +741,8 @@ def __pow__(self, power):
         return NotImplemented
 
     def __rpow__(self, base):
+        if self._is_parameterized_():
+            raise ValueError('Cannot raise to power when parameterized.')
```

Comment:
```suggestion
            raise NotImplementedError('Cannot raise to power when parameterized.')
```

Reply:
I did `return NotImplemented`, which seems to be the standard for dunders.